Sources/SwiftKafka/KafkaConsumerMessage.swift

```diff
@@ -0,0 +1,78 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the swift-kafka-gsoc open source project
+//
+// Copyright (c) 2022 Apple Inc. and the swift-kafka-gsoc project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of swift-kafka-gsoc project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Crdkafka
+import struct Foundation.Data
+
+/// Messages that are received from the Kafka cluster.
+public struct KafkaConsumerMessage {
+    let topic: String
+    let partition: Int32
+    let key: Data?
```

Comment:
According to [the proposal](https://github.com/felixschlegel/ProposalSwiftGSoC22/blob/main/main.pdf), we wanted `key` and `value` to be `ContiguousBytes`. However, we are just reading this `struct` and not modifying anything. Wouldn't it be more convenient to go with a concrete implementation of `ContiguousBytes` here such as `Data`?